interfaces/Portal/src/app/program/table-filter-row/table-filter-row.component.html

```diff
@@ -12,13 +12,17 @@
       class="util--flex-row ion-align-items-center"
       style="flex-basis: 80%"
     >
-      <span>
+      <span
+        class="ion-align-items-center ion-text-right"
+        style="width: 9rem"
+      >
         <ion-icon
           name="funnel"
           color="black"
           aria-hidden="true"
-          class="ion-margin-start"
         ></ion-icon>
+
+        {{ 'page.program.program-people-affected.filter-label' | translate }}
```

Comment:
I think this text-labels makes more sense if it sits 'next to': `page.program.table-filter-row.select-column-to-filter`, right?

So inside the `table-filter-row`-group.

But this kind of issues is also a good candidate for a more helpful/specific guideline in the wiki :)